contracts/eosio.system/src/limit_auth_changes.cpp

```diff
@@ -0,0 +1,50 @@
+#include <eosio.system/limit_auth_changes.hpp>
+#include <eosio.system/eosio.system.hpp>
+
+namespace eosiosystem {
+
+   void system_contract::limitauthchg(const name& account, const std::vector<name>& allow_perms,
+                                      const std::vector<name>& disallow_perms) {
```

Comment:
This allows`active` to restrict `owner`.  Even though `owner` can restore its own permissions, it still seems odd.